pkg/evaluator/builtin.go

```diff
@@ -94,6 +95,38 @@ func hasFunc(args []Value) Value {
 	return &Bool{Val: ok}
 }
 
+var delDecl = &parser.FuncDecl{
+	Name: "del",
+	Params: []*parser.Var{
+		{Name: "map", T: parser.ANY_TYPE},
```

Comment:
some comments here as for `has`, below too in the same way.

Reply:
ok, done.